### PR TITLE
Feature/child info progress, made info progress text, updated ChildInfo to receive props

### PR DESCRIPTION
**Pull request by jrgoodpasture**
…ress]

### What was the feature/problem?

{Please write here}

### How does this PR address the above feature/problem?

{Please write here}

### Check lists (check `x` in `[ ]` of list items)

- [ x] Test passed
- [ ] Coding style (indentation, etc)

### Additional Comments (if any)

{Please write here}

### CODE REVIEW

File: ui/src/childinfo/ChildInfo.js [hunk 1]
```diff
@@ -6,10 +6,9 @@ import './ChildInfo.css';
 export default class ChildInfo extends Component {
   constructor(props){
       super();
```
Comment (aswin1689) [inline ui/src/childinfo/ChildInfo.js]: don't we need to pass in props in super?

File: ui/src/childinfo/ChildInfo.js [hunk 2]
```diff
@@ -19,27 +18,24 @@ export default class ChildInfo extends Component {
...
       }
+      this.numSteps = Object.keys(this.stepMapFunction()).length
     }
 
     stepMapFunction = () => {
       return {
         0: {
-          text: 'Hi what is your name?',
-          input: 'name'
+          text: `Hi${this.props.name ? ` ${this.props.name}` : ''}, I have a few questions for you before we can make your wish come true!`,
```
Comment (aswin1689) [inline ui/src/childinfo/ChildInfo.js]: can we destructure name prop to be consistent?

Comment (jrgoodpasture) [reply ui/src/childinfo/ChildInfo.js]: Sure Ill do that

File: ui/src/childinfo/ChildInfo.js (hunk 1 above)
Comment (jrgoodpasture) [reply ui/src/childinfo/ChildInfo.js]: Not totally sure but Ill add that in

**Pull request merged**